tests/docker/README.md

```diff
@@ -8,13 +8,35 @@ Running Tests
 
 To run the tests first make sure you have the code that is to be tested checked out.
 
-Run the tests by executing the following in this directory:
+Build the Docker images by executing the following in this directory:
 
 ```
 docker-compose build
-docker-compose run globus
 ```
 
+Then run one or more of the following to execute the test on the Globus server of your choice.
+
+``` 
+docker-compose run globus-ubuntu20
+docker-compose run globus-ubuntu22
+docker-compose run globus-ubuntu24
+docker-compose run globus-el8
+docker-compose run globus-el9
+docker-compose run globus-debian11
+docker-compose run globus-debian12
+```
+Running Tests Against Unreleased Versions of iRODS
+--------------------------------------------------
+
+To run the tests against unreleased versions of iRODS, first the iRODS packages need to be built and stored in a local directory. The [iRODS Development Environment](https://github.com/irods/irods_development_environment) can be used to build the packages for each of the OS's.
+
+After this is done, make the changes to the following files to use these built packages:
+
+1. Update `docker-compose.yml` and uncomment the `/local/path/to/<os>/packages:/irods_packages` lines.  For the iRODS server entry, you will also need to uncomment the `volumes:` line.
+2. For each of the `globus.<os>.Dockerfile` files, comment out the lines that install the irods-icommands and irods-dev(el) packages.
```

Comment:
Is it possible that these can be left in and then the unreleased packages would upgrade them? Or does that not work?

Reply:
That's fair. We can't downgrade packages, so we can just leave these instructions for now.